server/mailer.js

```diff
@@ -204,12 +204,18 @@ export class Mailer {
   };
 
   signin = async (opts: { to: string, token: string, teamUrl: string }) => {
+    const signInLink = signinEmailText(opts);
+
+    if (process.env.NODE_ENV === "development") {
+      Logger.info(`Sign-In link: ${signInLink}`);
+    }
```

Comment:
I can go either way with leaving this in, but it's useful when the dev mail service is unavailable or if the developer doesn't have a network connection